<launch>
    <node name="visual_nav" pkg="mission" type="mission" output="screen"/>
</launch>